content/en/docs/languages/java/automatic/configuration.md

```diff
@@ -17,16 +17,7 @@ to find settings such as configuring export or sampling.
 Here are some quick links into those docs for the configuration options for
 specific portions of the SDK & agent:
 
-- [Exporters](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters)
-  - [OTLP exporter (span, metric, and log exporters)](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#otlp-exporter-span-metric-and-log-exporters)
-  - [Jaeger exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#jaeger-exporter)
-  - [Zipkin exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#zipkin-exporter)
-  - [Prometheus exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#prometheus-exporter)
-  - [Logging exporter](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#logging-exporter)
-- [Trace context propagation](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#propagator)
-- [OpenTelemetry Resource and service name](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#opentelemetry-resource)
 - [Batch span processor](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure/README.md#batch-span-processor)
```

Comment:
You seem to have not brought over certain sections from the SDK. Why include some content here and exclude others? I see the explanation in the PR description:

> I left links out to Batch span processor, Span limits, and Using the SPI to further configure the SDK since I didn't see similar sections in other languages, but let me know if we want all of them ported over and I can do those as well

But this seems wrong. We should document all of the SDK environment variables in one place. If opentelemetry.io is that place, it needs to be complete.

Reply:
I included everything now